iterative/utils/analytics.go

```diff
@@ -223,26 +233,59 @@ func JitsuEventPayload(action string, e error, d *schema.ResourceData) map[strin
 	return payload
 }
 
-func SendJitsuEvent(action string, e error, d *schema.ResourceData) {
-	if d == nil {
-		return
+func SendJitsuEvent(action string, e error, extra map[string]interface{}) {
+	for _, prefix := range []string{"ITERATIVE", "DVC"} {
+		if _, ok := os.LookupEnv(prefix + "_NO_ANALYTICS"); ok {
+			logrus.Debugf("analytics: %s_NO_ANALYTICS environment variable is set", prefix)
+			return
+		}
 	}
 
-	if _, ok := os.LookupEnv("DO_NOT_TRACK"); ok {
-		logrus.Debug("Analytics: DO_NOT_TRACK enabled")
-		return
-	}
```

Comment:
I prefer the well known `DO_NOT_TRACK` environment variable. If any change should be done our products should use it.

Reply:
DVC uses `DVC_NO_ANALYTICS`, and in the specification we have `ITERATIVE_NO_ANALYTICS` as a general alternative. Disclaimer: it was me who wrote it. 😉

Reply:
No strong opinions on naming, but please discuss this at the specification level so all the products use the same name.

Reply:
yes `DO_NOT_TRACK` is [well known ](https://www.google.com/search?q=site%3Agithub.com+DO_NOT_TRACK)

Reply:
https://consoledonottrack.com